tools/pyboard.py

```diff
@@ -300,7 +307,28 @@ def __init__(
                             self.serial.rts = False  # RTS False = EN High = MCU enabled
                         self.serial.open()
                     else:
-                        self.serial = serial.Serial(device, **serial_kwargs)
+                        #self.serial = serial.Serial(device, **serial_kwargs)
```

Comment:
Nice find on the glitch on the rts lines on existing port open, this old commented it code line should be removed though.